docs/docs/contextual-conversations.mdx

```diff
@@ -1,120 +1,182 @@
-  - slot{"onboarding": true}
-  - utter_getstarted_new
-  - utter_built_bot_before
+A conversation with a returning user:
+```yaml
+User: How can I get started with Rasa?
+    Sara: To determine how I can help you best, I'm going to ask you a few questions.
+    Sara: Let's go. Are you new to Rasa?
+User: no
+    Sara: Ok, which product would you like to know more about? Rasa Open Source or Rasa X?
 ```
 
-For this to work, keep in mind that the slot has to be featurized in your `domain.yml`
-file. This time we can use the `text` slot type, as we only care about whether the
-[slot was set or not](./domain.mdx#slots/).
+This page is a guide on creating contextual conversation patterns using Rasa.
+
+## Contextual Conversation Patterns
+
+### 1. Creating Stories
 
-## AugmentedMemoizationPolicy
+[Stories](./stories.mdx) are used to train the [`MemoizationPolicy`](policies.mdx#memoizationpolicy) and machine
+learning policies like the [`TEDPolicy`](policies.mdx#tedpolicy). For example, in order for Sara to respond differently for new users
+and users with previous experience, you'd need stories like this:
 
-To make your bot more robust to interjections, you can replace the MemoizationPolicy
-with the AugmentedMemoizationPolicy. It works the same way as the MemoizationPolicy,
-but if no exact match is found it additionally has a mechanism that forgets a certain
-amount of steps in the conversation history to find a match in your stories (read more
-[here](policies.mdx#augmented-memoization-policy))
 
-## Using ML to generalise
+```yaml
+stories:
+  - story: New user
+    steps:
+  - intent: how_to_get_started
+  - action: utter_getstarted
+  - action: utter_first_bot_with_rasa
+  - intent: affirm
+  - action: action_set_onboarding
+  - slot_was_set:
+    - onboarding: true
+  - action: utter_built_bot_before
+
+  - story: Not new user
+    steps:
+  - intent: how_to_get_started
+  - action: utter_getstarted
+  - action: utter_first_bot_with_rasa
+  - intent: deny
+  - action: action_set_onboarding
+  - slot_was_set:
+    - onboarding: false
+  - action: utter_ask_which_product
+```
+
+These stories diverge based on the users intent (`affirm` or `deny`). Using
+on the user's intent, an action sets a slot that further directs
+the conversation.
+
+### 2. Defining Slots
+
+[Slots](domain.mdx#slots) are your assistant's memory. Slots store pieces of information that your
+assistant needs to refer to later and can direct the flow of the conversation
+based on `slot_was_set` events. There are different [types of slots](domain.mdx#slot-types),
+and each affects the conversation flow in its own way.
+
+In the getting started example, the `onboarding` slot is a boolean slot. If it is true, the
+user is treated as a new user, and if it is false, as a user with previous bot building experience.
+You define a slot and its type in the domain:
+
+```rasa-yaml
+slots:
+  onboarding:
+    type: bool
+```
+
+### 3. Configuring the `TEDPolicy`
```

Comment:
why is this called configuring the TEDPolicy? There's nothing about configuration in here